lib/Db/Face.php

```diff
@@ -138,18 +138,6 @@ public static function fromModel(int $image, array $faceFromModel): Face {
 		return $face;
 	}
 
-	/**
-	 * Helper method, to normalize face sizes back to original dimensions, based on ratio
-	 *
-	 * @param float $ratio Ratio of image resize
-	 */
```

Comment:
Excellent for removing this (refactoring this whole normalize code! it looks better now!)